manifests/config.pp

```diff
@@ -34,6 +34,12 @@
         }
       }
       'systemd' : {
+        file { '/etc/sysconfig/nomad':
+          ensure  => directory,
```

Comment:
hi 
i can't see any fail in indentation on line 38-41, can you give more details? it started with 2 space and 4 after class declaration. is it not the right way ?
Regards
Karen